spark-testing/spark_utils.py

```diff
@@ -27,7 +27,11 @@
 SPARK_SERVICE_ACCOUNT_SECRET = os.getenv("SPARK_SERVICE_ACCOUNT_SECRET", "spark-service-acct-secret")
 SPARK_SERVICE_NAME = os.getenv("SPARK_SERVICE_NAME", "spark")
 FOLDERED_SPARK_SERVICE_NAME = "/path/to/" + SPARK_SERVICE_NAME
+
 SPARK_USER = os.getenv("SPARK_USER", "nobody")
+SPARK_HISTORY_USER = os.getenv("SPARK_USER", SPARK_USER)
```

Comment:
There is never a way that these could differ (unless they are overwritten after executing these lines):
```suggestion
SPARK_HISTORY_USER = SPARK_USER
```

Reply:
good catch, fixed